src/test/unit/process-profile.test.js

```diff
@@ -592,6 +592,32 @@ describe('gecko samples table processing', function () {
   });
 });
 
+describe('threadCPUDelta processing', function () {
+  it('removes threadCPUDelta data when the array is filled with null values', () => {
```

Comment:
Could it be good to add a test with a single non-null value to check that we don't remove it? It would be nice to make sure we don't regress that case?

Reply:
sure